docs/adr/0006-url-encode-path-parameters.md

```diff
@@ -0,0 +1,51 @@
+# 0006 - URL-encode path parameters that contain forward slashes
+
+2023-03-16
+
+## Status
+
+Accepted
+
+## Context
+
+Users must be able to search the system for nominals on Police National Computer (PNC) ID. PNC ID is a unique person
+identifier. RESTful API design convention is that path parameters are used to identify a specific resource or resources.
+However, passing PNC IDs into our API as path parameters is problematised by the fact that PNC IDs contain a forward
+slash (e.g., 12/345345B) which is the URO standard path delimiter.
+
+We considered two options for dealing with the forward slash in PNC ID:    
+**1. Encode the PNC ID**
+
+- The PNC ID 12/345345B would be passed to the /persons end-point URL-encoded as 12%2F345345B.
+
+Considerations:
+
+- This would result in some added complexity for the consumer, who would have to encode the PNC ID.
```

Comment:
We also need to make sure our documentation clearly indicates that it's encoded and it's on them to do the encoding.

Reply:
I see you have this as a consequence, this can be ignored.